testsuite/classlibrary/TestPprotect.sc

```diff
@@ -0,0 +1,19 @@
+
+TestPprotect : UnitTest {
+	test_PprotectRecursion {
```

Comment:
please rename according to recommendations here - https://gist.github.com/brianlheim/91222d487afa18582c287b0a722ae272#naming-test-methods

the rest looks good!

Reply:
Done.